src/content/1.7/modules/core-updates/1.7.7.md

```diff
@@ -7,6 +7,113 @@ aliases:
+The following dependencies have been updated in order to provide support for PHP 7.3:
+
+Extension | Old version | New version
+---|---|---
+beberlei/doctrineextensions | v1.1.9 | v1.2.6
+behat/transliterator | v1.2.0 | v1.3.0
+composer/ca-bundle | 1.1.4 | 1.2.6
+composer/installers | v1.6.0 | v1.7.0
+composer/semver | 1.5.0 | 1.5.1
+composer/xdebug-handler | 1.3.2 | 1.4.0
+doctrine/annotations | v1.4.0 | v1.8.0
+doctrine/cache | v1.6.2 | 1.10.0
+doctrine/collections | v1.4.0 | 1.6.4
+doctrine/common | v2.5.3 | 2.12.0
+doctrine/dbal | v2.5.13 | v2.9.3
+doctrine/doctrine-bundle | 1.10.2 | 1.12.16
+doctrine/doctrine-cache-bundle | 1.3.5 | 1.4.0
+doctrine/inflector | v1.1.0 | 1.3.1
+doctrine/instantiator | 1.0.5 | 1.3.0
+doctrine/lexer | v1.0.1 | 1.0.2
+doctrine/orm | v2.5.14 | v2.7.0
+ezyang/htmlpurifier | v4.10.0 | v4.12.0
+fig/link-util | 1.0.0 | 1.1.0
+friendsofphp/php-cs-fixer | v2.15.1 | v2.16.1
+friendsofsymfony/jsrouting-bundle | 2.4.0 | 2.5.3
+greenlion/php-sql-parser | 4.2.1 | 4.3.0
+guzzlehttp/guzzle | 5.3.3 | 5.3.4
+johnkary/phpunit-speedtrap | v1.1.0 | v3.1.0
+league/tactician-bundle | v0.4.1 | v1.2.0
+markbaker/matrix | 1.1.4 | 1.2.0
+maxmind-db/reader | v1.4.1 | v1.6.0
+maxmind/web-service-common | v0.5.0 | v0.6.0
+mikey179/vfsstream | v1.6.5 | v1.6.8
+mobiledetect/mobiledetectlib | 2.8.33 | 2.8.34
+monolog/monolog | 1.24.0 | 1.25.3
+myclabs/deep-copy | 1.7.0 | 1.9.4
+nikic/php-parser | v3.1.5 | v4.3.0
+pear/archive_tar | 1.4.6 | 1.4.9
+pear/console_getopt | v1.4.2 | v1.4.3
+pear/pear-core-minimal | v1.10.9 | v1.10.10
+pear/pear_exception | v1.0.0 | v1.0.1
+pelago/emogrifier | v2.1.1 | v2.2.0
+phpdocumentor/reflection-common | 1.0.1 | 2.0.0
+phpdocumentor/reflection-docblock | 3.3.2 | 4.3.4
+phpdocumentor/type-resolver | 0.4.0 | 1.0.1
+phpoffice/phpspreadsheet | 1.7.0 | 1.10.1
+phpspec/prophecy | 1.8.0 | 1.10.1
+phpunit/php-code-coverage | 4.0.8 | 6.1.4
+phpunit/php-file-iterator | 1.4.5 | 2.0.2
+phpunit/php-timer | 1.0.9 | 2.1.2
+phpunit/php-token-stream | 1.4.12 | 3.1.1
+phpunit/phpunit | 5.7.27 | 7.5.20
+psr/log | 1.1.0 | 1.1.2
+sebastian/comparator | 1.2.4 | 3.0.2
+sebastian/diff | 1.4.3 | 3.0.2
+sebastian/environment | 2.0.0 | 4.2.3
+sebastian/exporter | 2.0.0 | 3.1.2
+sebastian/global-state | 1.1.1 | 2.0.0
+sebastian/object-enumerator | 2.0.1 | 3.0.3
+sebastian/recursion-context | 2.0.0 | 3.0.0
+sebastian/resource-operations | 1.0.0 | 2.0.1
+sensio/distribution-bundle | v5.0.24 | v5.0.25
+sensio/framework-extra-bundle | v5.2.4 | v5.4.1
+sensiolabs/security-checker | v5.0.3 | v6.0.3
+smarty/smarty | v3.1.33 | v3.1.34
+swiftmailer/swiftmailer | v5.4.12 | v6.2.3
+symfony/monolog-bundle | v3.3.1 | v3.5.0
+symfony/phpunit-bridge | v3.4.31 | v3.4.36
+symfony/swiftmailer-bundle | v2.6.7 | v3.2.6
+symfony/symfony | v3.4.26 | v3.4.37
+tecnickcom/tcpdf | v6.2.26 | 6.3.2
+tijsverkoyen/css-|-inline-styles | 2.2.1 | 2.2.2
+twig/twig | v1.38.4 | v1.42.4
+webmozart/assert | 1.4.0 | 1.6.0
+
+## New hooks
+
+The following hooks have been added:
```

Comment:
Do you want us to put here all new 1.7.7. hooks ? @tdavidsonas88 is working on a similar task

Reply:
Yes, and also in the "all hooks" page